<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>√Ålgebra Bustamante</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="matrix_listWidget"/>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Operands</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>op1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>op2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Generate</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Generate</string>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>row</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="row_spinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>col</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="col_spinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>block row</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="block_row_spinBox">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>block col</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="block_col_spinBox">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QPushButton" name="generate_pushButton">
                 <property name="text">
                  <string>Generate Matrix</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Gauss</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Gaussian Elimination</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QCheckBox" name="checkBox">
                 <property name="text">
                  <string>Vertical Pivot</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="text">
                  <string>Horizontal Pivot</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="exec_gauss_pushButton">
                 <property name="text">
                  <string>Execute</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>matrix_listWidget</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>verticalLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
